.github/workflows/devtools.yml

```diff
@@ -0,0 +1,81 @@
+name: DevTools
+
+on:
+  push:
+    branches: [master]
+  pull_request: # run on all PRs, not just PRs to a particular branch
+
+jobs:
+  smoke-devtools:
+    runs-on: macos-latest
```

Comment:
Is running on a mac important? I've heard they're slower than the other bots (but I have no evidence).

Reply:
They are definitely slower. I ran extensive macbot testing for awhile on private repos and disabling yarn caching was actually a massive improvement because unzipping the archive took ~2x longer than streaming from network.

Reply:
Linux crashes.